src/validators/cache/index.js

```diff
@@ -1,6 +1,14 @@
-import {noop} from 'lodash'
+import {friendlyTypeOf} from '../../utils'
+
 export default {
   key: 'cache',
-  validate: noop,
+  validate: validateCache,
+}
+
+function validateCache(value) {
+  const type = friendlyTypeOf(value)
+  if (type !== 'boolean' && type !== 'object') {
+    return `Unexpected type ${type}. Must be a boolean or an object.`
```

Comment:
Apart from that: 👍🏻. Will hit the green button as soon as this is done so you can use the shiny util fn in the other PR. ;)